avocado/core/test.py

```diff
@@ -797,9 +791,9 @@ def _run_avocado(self):
         whiteboard_file = os.path.join(self.logdir, 'whiteboard')
         genio.write_file(whiteboard_file, self.whiteboard)
 
-        if self.job is not None:
-            output_check_record = self.job.config.get('run.output_check_record')
-            output_check = self.job.config.get('run.output_check')
+        if self.__config is not None:
+            output_check_record = self.__config.get('run.output_check_record')
```

Comment:
With my previous suggestion, `self.__config` will always have a dict even empty, right? So maybe this if is unecessary.